modules/backend/formwidgets/relation/assets/less/relation.css

```
@@ -0,0 +1,55 @@
+div[data-control="relation"] .field-checkboxlist .checkboxlist-controls > div:nth-child(even) {
+  margin-left: auto;
+  margin-right: 0;
+}
+div[data-control="relation"] .checkboxlist-item {
+  --background-padding: 10px;
+  display: flex;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)